FFXIVClientStructs/FFXIV/Client/System/Timer/ClientTime.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FFXIVClientStructs.FFXIV.Client.System.Timer;
+
+[StructLayout(LayoutKind.Explicit)]
+public struct ClientTime {
+    [FieldOffset(0x8)] public uint Time; //Timestamp
+    [FieldOffset(0xC)] public uint UnkC;
+}
```

Comment:
I guess I can suggest these changes too. 😄

```suggestion
// Client::System::Timer::ClientTime
//   Common::Game::Time::GameTime
[StructLayout(LayoutKind.Explicit, Size = 0x40)]
public struct ClientTime {
    [FieldOffset(0x8)] public long EorzeaTime;
    [FieldOffset(0x10)] public float EorzeaTimeRemainder;

    [FieldOffset(0x30)] public long EorzeaTimeOverride;
    [FieldOffset(0x38)] public bool IsEorzeaTimeOverridden;
}
```